docs/fides/docs/guides/generate_resources.md

```diff
@@ -367,3 +367,74 @@ Successfully scanned the following systems:
 
 Resource coverage: 100%
 ```
+
+## Working With a GCP Account
+
+The `generate` command can connect to a GCP account and automatically generate resource YAML files based on tracked resources. Currently, generating datasets from BigQuery is supported.
+
+### Providing Credentials
+
+GCP credentials can be generated via a [service account keyfile](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating) which can be passed as a command option or the fides config. You will need to set project specific credentials for access rights, but datasets can be passed explicitly at runtime.
+
+#### Command Options
+
+The path to the keyfile can be directly supplied in your command using the `keyfile_path` option.
+```sh
+...
+--keyfile-path "/path/to/keyfile.json"
+...
+```
+
+#### Fides Config
+Credentials can be defined within your [fides config](../installation/configuration.md) under the credentials section.
+
+```sh
+my_gcp_credentials = {type = "service_account", project_id = "<my_project_id>", private_key_id = "<my_private_key_id>", private_key = "<my_private_key>", client_email = "<my_client_email>", client_id = "<my_client_id>", auth_uri = "https://accounts.google.com/o/oauth2/auth", token_uri = "https://oauth2.googleapis.com/token", auth_provider_x509_cert_url = "https://www.googleapis.com/oauth2/v1/certs", client_x509_cert_url = "<my_cert_url>"}
+```
```

Comment:
I would consider using the same formatting from our test config file:

```
[credentials.bigquery_1]
type = "service_account"
project_id = "redacted_id_override_in_tests"
private_key_id = "redacted_id_override_in_tests"
private_key = "redacted_id_override_in_tests"
client_email = "redacted_id_override_in_tests"
client_id = "redacted_id_override_in_tests"
auth_uri = "https://accounts.google.com/o/oauth2/auth"
token_uri = "https://oauth2.googleapis.com/token"
auth_provider_x509_cert_url = "https://www.googleapis.com/oauth2/v1/certs"
client_x509_cert_url = "redacted_url_override_in_tests"
```

Reply:
ya great catch @earmenda - thanks for pointing this out 🙌🏽